src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,330 @@
+    public static final String EMPTY_DEADLINE_DESCRIPTION = "OOPS!!! The description of a deadline cannot be empty.";
+    public static final String EMPTY_EVENT_DESCRIPTION = "OOPS!!! The description of a event cannot be empty.";
+    public static final String INVALID_INPUT = "This is an invalid input, please follow this input format\n";
+    public static final String DEADLINE_INVALID_INPUT = INVALID_INPUT + DEADLINE_FORMAT;
+    public static final String EVENT_INVALID_INPUT = INVALID_INPUT + EVENT_FORMAT;
+    public static final String MEANINGLESS_SENTENCE_AFTER_KEYWORD = "OPPS!!! The sentence after keyword has no meaning";
+    public static final String REMOVE_MESSAGE = "Noted. I've removed this task:";
+    public static final String WRITEFILE_EXCEPTION_MESSAGE = "OPPS!!! Something went wrong when you write to data file";
+    public static final ArrayList<String> KEYWORDS = new ArrayList<>(
+            List.of("todo","deadline","event","list","bye","mark","unmark","help","delete")
+    );
+
+    public static void main(String[] args) {
+        ArrayList<Task> tasks = new ArrayList<>();
+        FileClass dataFile = new FileClass("src/main/data.txt", tasks);
+        try {
+            dataFile.addTasks();
+        } catch (FileNotFoundException e) {
+            System.out.println("Please give a correct data file path");
+            return;
+        }
+        showWelcomeMessage();
+
+        String userInput;
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            userInput = getInput(in);
+            String[] separatedKeyWordAndContent = userInput.split(" ", 2);
+            String keyword = separatedKeyWordAndContent[0];
+            if (!KEYWORDS.contains(keyword)) {
+                System.out.println(INCORRECT_KEYWORD);
+                continue;
+            }
+            if (userInput.equals("bye")) {
+                System.out.println(BYE_MESSAGE);
+                createHorizontalLine();
+                break;
+            } else if (userInput.equals("help")) {
+                showHelpMessage();
+            } else if (userInput.equals("list")) {
+                printItems(tasks);
+            } else if (keyword.equals("delete")) {
+                if (userInput.split(" ").length != 2) {
+                    System.out.println(INSTRUCTION + "\n"
+                            + keyword + ": Number");
+                } else {
+                    deleteTask(tasks, separatedKeyWordAndContent, dataFile);
+                }
+            } else if (keyword.equals("todo")) {
+                if (doesIndexOutOfBoundsOccur(separatedKeyWordAndContent,1, EMPTY_TODO_DESCRIPTION)) {
+                    continue;
+                }
+                try {
+                    addTodoTask(separatedKeyWordAndContent[1], tasks, dataFile);
+                } catch(EmptyDescription e) {
+                    System.out.println("OOPS!!! your task can not be empty");
+                }
```

Comment:
Having these if-else structures here causes two issues: making your code too long and causing it closer to an arrowhead code. Consider refactoring them out for better readability.